### DISCUSSION TITLE
HD Radio decoder (NRSC5)

### DISCUSSION THREAD

**Discussion by craigerl** [AlexandreRouma/SDRPlusPlus#1450] [Ideas]
High definition radio is popular in the americas which includes a side-band PSK digital signal for many wide-band FM broadcast stations.

NRSC5 decoder is open source at https://github.com/theori-io/nrsc5

-craig

**Comment by AlexandreRouma** [comment]
Several things:
1) All modules I publish and SDR++ in general have to be written from scratch, I refuse to copy anyone's code or even use other people's code as documentation.
2) The codec used for HD-Radio is proprietary and I'm not trying to get a lawsuit at the moment.

If someone else wants to make a third party module and risk getting sued by iBiquity in the process, they are free to do so.
